components/ledger/pkg/storage/sqlstorage/ledger/migrates/0-init-schema/postgres.sql

```diff
@@ -31,6 +31,20 @@ CREATE FUNCTION "VAR_LEDGER_NAME".use_account(postings jsonb, account character
     LANGUAGE sql
     AS $$ SELECT bool_or(v.value) from ( SELECT "VAR_LEDGER_NAME".use_account_as_source(postings, account) AS value UNION SELECT "VAR_LEDGER_NAME".use_account_as_destination(postings, account) AS value ) v $$;
 
+--statement
```

Comment:
Can you use sql function instead of pgsql ?
pgsql has a big cost.

Reply:
Done thanks ! I used coalesce like you said !